docs/public-networks/how-to/connect/manage-peers.md

```diff
@@ -83,6 +84,8 @@ With discovery disabled, peers can't open connections with the node unless they
 
 If your nodes fail to connect, ensure the [required ports are open](configure-ports.md).
 
-If your nodes are running in AWS, check you have appropriate `SecurityGroups` to allow access to the required ports.
+If your nodes are running in AWS, ensure you have the appropriate `SecurityGroups` configured to allow access to the required ports.
 
 Check that the [enode URLs](../../concepts/node-keys.md#enode-url) specified for [bootnodes](../../../private-networks/how-to/configure/bootnodes.md) or [static nodes](static-nodes.md) match the enode URLs displayed when starting the remote nodes.
+
+If your node is having trouble peering, see the [troubleshoot peering documentation](../../how-to/troubleshoot/peering.md).
```

Comment:
```suggestion
If you still have peering issues, see the [troubleshoot peering documentation](../../how-to/troubleshoot/peering.md).
```

This page is about peering, and so this section is already about troubleshooting peering. So the additional troubleshoot peering page is for issues not fixed by these first steps. Alternatively, we could consolidate this section into the troubleshoot peering page and just link to that.

Reply:
Yes, it would probably be best practice to add this section to the troubleshooting page and just link to the separate page.